src/libs/Navigation/PlatformStackNavigation/navigationOptions/animation/fade/index.ts

```diff
@@ -0,0 +1,7 @@
+import type {NativeStackNavigationOptions} from '@react-navigation/native-stack';
+import type {PlatformSpecificNavigationOptions} from '@libs/Navigation/PlatformStackNavigation/types';
+import Animations from '..';
+
+const fade: PlatformSpecificNavigationOptions = {animation: Animations.FADE, animationDuration: 150} satisfies NativeStackNavigationOptions;
```

Comment:
> Or rather, what is the current animation/transition duration?

I think at the moment it's 500ms, but visual perception is about 350ms (it's a spring animation and last 150ms we may just animate opacity from 0.95 to 1, which might be not an actual duration perception).

Do you want me to change it to 100ms?

Reply:
@shawnborton @dannymcclain @dubielzyk-expensify I will now go ahead with this change, we can play around with the delay based on how it will feel later